app/models/item.rb

```diff
@@ -22,10 +22,10 @@ class Item < ApplicationRecord
   has_many :requests, dependent: :destroy
   has_many_attached :photos
 
-  # Validations
-  validates_presence_of :user_id, :photos, :description, :expiration_date, :item_type, :status, :name
+  validates_presence_of :user_id :description, :expiration_date, :item_type, :status, :name
   validates :description, length: { minimum: 7 }
```

Comment:
@MarlAndre please add back `:photos` in as a validation check, it was a recent change pushed to master by Denzel.

Reply:
agreed